connect/runtime/src/test/java/org/apache/kafka/connect/runtime/WorkerSinkTaskTest.java

```diff
@@ -1999,6 +2004,81 @@ public void testErrorReporterConfigurationExceptionPropagation() {
         PowerMock.verifyAll();
     }
 
+    @Test
+    public void testPartitionCountInCaseOfPartitionRevocation() {
+        mockConsumer = new MockConsumer<>(OffsetResetStrategy.EARLIEST);
```

Comment:
If we make this a local variable, we can change the left-hand type to `MockConsumer<byte[], byte[]>` and then the casts later on at lines 2019 and 2022 won't be necessary.

Reply:
Yes, I missed this, Thanks, I have added this change.